src/styles/main.scss

```diff
@@ -0,0 +1,62 @@
+$dial-size: 80vmin;
+$dial-center-size: $dial-size * 0.1;
+
+@keyframes rotate360 {
+  from {
+    transform: rotate(0);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.body {
+  margin: 0;
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.dial {
+  position: absolute;
+  border: 4px dotted black;
+  width: $dial-size;
+  height: $dial-size;
+  border-radius: 50%;
+
+  &__center {
+    width: $dial-center-size;
+    height: $dial-center-size;
+    border-radius: 50%;
+    background-color: slategrey;
+    text-align: center;
+    position: absolute;
+    top: 47%;
+    left: 46.5%;
+  }
+
```

Comment:
It's better to add a top and left position with the correct percentage value (50%) and use 'transform: translate(-50%, -50%)' to center the element precisely.